frontstage/templates/message-error-summary.html

```diff
@@ -10,6 +10,12 @@ <h1 class="panel__title venus">There are {{ errors|length }} errors on this page
                 {% endif %}
             </div>
             <div class="panel__body" data-qa="error-body">
+                {% if errors|length == 1 %}
+                <p>This <b>must be corrected </b>to continue.</p>
```

Comment:
I think the standard is not to, it's more important to keep the html elements in line